health/health_json.c

```diff
@@ -76,7 +82,22 @@ void health_alarm_entry2json_nolock(BUFFER *wb, ALARM_ENTRY *ae, RRDHOST *host)
                    , (ae->flags & HEALTH_ENTRY_FLAG_SILENCED)?"true":"false"
     );
 
-    health_string2json(wb, "\t\t", "info", ae->info?ae->info:"", ",\n");
+    char *replaced_info = NULL;
+    if (likely(ae->info)) {
+        char *m = NULL;
+        replaced_info = strdupz(ae->info);
+        size_t pos = 0;
+        while ((m = strstr(replaced_info + pos, "$family"))) {
+            char *buf = NULL;
+            pos = m - replaced_info;
+            buf = find_and_replace(replaced_info, "$family", ae->family, m);
```

Comment:
may want to check for ae->family null here as in other places

Reply:
Thanks Stel. I've left it unchecked, because it's also unchecked a few lines above when passed to the buffer. I though that because of that it should be taken care of. But yes, you are right, I can check there for null and pass empty string to be replaced.

In the other 2 occurrences of this, the value is checked the same way it is checked later in the code.

Reply:
OK, thanks, fixed with https://github.com/netdata/netdata/pull/10961/commits/a6e1d469795f3824a9b7c9798df7a21816bd7e57